dev_utilities.py

```diff
@@ -2,7 +2,6 @@
 
 try:
   import dev_appserver
-  dev_appserver.fix_sys_path()
```

Comment:
This line was throwing an error for me, google appengine versionitis?

Reply:
please leave this in, it allows dev_server.sh to work.   and probably doesn't require your PYTHONPATH mangling